LoginForm.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Formik, Form, Field } from 'formik';
+import { useHistory } from 'react-router-dom';
+
+function LoginForm() {
+  let history = useHistory();
+
+  return (
+    <Formik
+      initialValues={{ email: '', password: '' }}
+      onSubmit={(values, { setSubmitting }) => {
```

Comment:
Consider adding a validation schema to Formik for better form validation. You can use Yup to create the schema.